### PR TITLE
Simplifies introduction

### PR DESCRIPTION
**Pull request by tplooker**
Re-simplifies the introduction

### CODE REVIEW

**Review by selfissued — APPROVED** [review]
I agree with this direction.  I would try to eliminate places where the spec appears to be make normative statements about having a backend.  (It's fine to say things like "1. A Client Attestation JWT - typically produced by the client backend.")

**Review by c2bo — APPROVED** [review]

File: draft-looker-oauth-attested-key-based-client-authentication.md [hunk 1]
```diff
@@ -109,15 +109,9 @@ The following diagram depicts the overall architecture and protocol flow.
...
-
-The client can proceed now and generate a Client Attestation Proof of Possession (PoP) for the Client Instance Key (step 5). Lastly, the client sends both the Client Attestation JWT and the Client Attestation PoP with the Token Request to the Authorization Server. The Authorization Server validates the client attestation and thus authenticates the client. The Authorization Server may continue to issue sender-constrained access tokens using {{DPOP}}.
-
-Note that the protocol for steps 2 and 4 and how the client instance authenticates with the client backend is out of scope of this specification.
-
-This specification defines the format of the Client Attestation that a client instance uses to authenticate in its interactions with an authorization server, which is comprised of two key parts:
+This specification only defines the format of the client assertion that a client instance uses to authenticate in its interactions with an authorization server (indicated in step 6), which is comprised of two key parts:
 
 1. A Client Attestation JWT- produced by the client backend.
 2. A Client Attestation Proof of Possession (PoP) - produced by the client instance.
```
Comment (peppelinux) [inline draft-looker-oauth-attested-key-based-client-authentication.md:117]:
```suggestion
1. A Client Attestation JWT produced by the client backend.
2. A Client Attestation Proof of Possession (PoP) produced by the client instance.
```

**Review by peppelinux — CHANGES_REQUESTED** [review]
informally approved, formally requested two small editorial things

**Comment by tplooker** [comment]
Closed in favour of changes made in #52

**Pull request closed without merging**